controls/ccn_rhel9.yml

```
@@ -0,0 +1,836 @@
+      leastthe partial status after the templated rule.
+
+  - id: A.4.SEC-RHEL3
+    title: Permissions by partitions are modified
+    original_title: Se modifican los permisos por particiones.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      Related to nosuid, noexec and nodev options but in /boot. More context is needed.
+
+  #[A.5] Suplantación de la identidad.
+  - id: A.5.SEC-RHEL1
+    title: Control login and impersonation permissions
+    original_title: Se controlan los permisos de inicio de sesión y suplantación de identidad.
+    levels:
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - sudo_add_use_pty
+      - use_pam_wheel_for_su
+
+  - id: A.5.SEC-RHEL2
+    title: Elevation attempts are controlled by defining sudoers users and groups
+    original_title: Se controlan los intentos de elevación mediante definición de usuarios y grupos sudoers.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - sudo_require_authentication
+      - sudo_require_reauthentication
+
+  - id: A.5.SEC-RHEL3
+    title: Access to encryption keys is controlled
+    original_title: Se controla el acceso a las claves de cifrado.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: pending
+    notes: |-
+      There are rules for ssh_keys, for example. We need to confirm the scope of this requirement
+
+  - id: A.5.SEC-RHEL4
+    title: Disable insecure encryption algorithms
+    original_title: Se han deshabilitado los algoritmos de cifrado inseguros.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - configure_crypto_policy
+      - var_system_crypto_policy=default_policy
+
+  - id: A.5.SEC-RHEL5
+    title: Recurring password change required
+    original_title: Se exige el cambio de contraseña de forma recurrente.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - accounts_minimum_age_login_defs
+      - var_accounts_minimum_age_login_defs=2
+      - accounts_password_set_min_life_existing
+      - accounts_maximum_age_login_defs
+      - var_accounts_maximum_age_login_defs=45
+      - accounts_password_set_max_life_existing
+      - accounts_password_warn_age_login_defs
+      - var_accounts_password_warn_age_login_defs=10
+      - accounts_password_set_warn_age_existing
+
+  - id: A.5.SEC-RHEL6
+    title: Secure protocols are used for network authentication processes
+    original_title: Se hace uso de protocolos seguros para los procesos de autenticación de red.
+    levels:
+      - basic
+      - intermediate
+      - advanced
+    status: planned
+    related_rules:
+      - configure_ssh_crypto_policy
+
+  - id: A.5.SEC-RHEL7
+    title: Control network session inactivity
```

```suggestion
    title:Network session inactivity is controlled
```
Modified it for an increased consistency with the rest of the titles.